validator/src/cli.rs

```diff
@@ -1330,6 +1333,35 @@ pub fn app<'a>(version: &'a str, default_args: &'a DefaultArgs) -> App<'a, 'a> {
                        up to the default or specified total bytes in the \
                        ledger")
         )
+        .arg(
+            Arg::with_name("replaying_backend")
+                .long("replaying-backend")
+                .hidden(true)
+                .value_name("BACKEND")
+                .takes_value(true)
+                .possible_values(&[
+                    "blockstore-processor",
+                    "unified-scheduler",
+                    ])
+                .default_value(&default_args.replaying_backend)
+                .help(
+                    "Switch transaction scheduling backend for validating ledger entries"
```

Comment:
I carefully worded to be fit for `solana-validator`/`solana-ledger-tool` and `replaying`/`banking`.